stl/inc/chrono

```diff
@@ -617,27 +669,1290 @@ namespace chrono {
+    }
+    _NODISCARD constexpr year_month_day operator/(int _Year, const month_day& _Md) noexcept {
+        return year{_Year} / _Md.month() / _Md.day();
+    }
+    _NODISCARD constexpr year_month_day operator/(const month_day& _Md, const year& _Year) noexcept {
+        return _Year / _Md.month() / _Md.day();
+    }
+    _NODISCARD constexpr year_month_day operator/(const month_day& _Md, int _Year) noexcept {
+        return year{_Year} / _Md.month() / _Md.day();
+    }
+    _NODISCARD constexpr year_month_day_last operator/(const year_month& _Ym, last_spec) noexcept {
+        return {_Ym.year(), month_day_last{_Ym.month()}};
+    }
+    _NODISCARD constexpr year_month_day_last operator/(const year& _Year, const month_day_last& _Mdl) noexcept {
+        return {_Year, _Mdl};
+    }
+    _NODISCARD constexpr year_month_day_last operator/(int _Year, const month_day_last& _Mdl) noexcept {
+        return year{_Year} / _Mdl;
+    }
+    _NODISCARD constexpr year_month_day_last operator/(const month_day_last& _Mdl, const year& _Year) noexcept {
+        return _Year / _Mdl;
+    }
+    _NODISCARD constexpr year_month_day_last operator/(const month_day_last& _Mdl, int _Year) noexcept {
+        return year{_Year} / _Mdl;
+    }
+    _NODISCARD constexpr year_month_weekday operator/(const year_month& _Ym, const weekday_indexed& _Wdi) noexcept {
+        return year_month_weekday{_Ym.year(), _Ym.month(), _Wdi};
+    }
+    _NODISCARD constexpr year_month_weekday operator/(const year& _Year, const month_weekday& _Mwd) noexcept {
+        return year_month_weekday{_Year, _Mwd.month(), _Mwd.weekday_indexed()};
+    }
+    _NODISCARD constexpr year_month_weekday operator/(int _Year, const month_weekday& _Mwd) noexcept {
+        return year{_Year} / _Mwd;
+    }
+    _NODISCARD constexpr year_month_weekday operator/(const month_weekday& _Mwd, const year& _Year) noexcept {
+        return _Year / _Mwd;
+    }
+    _NODISCARD constexpr year_month_weekday operator/(const month_weekday& _Mwd, int _Year) noexcept {
+        return year{_Year} / _Mwd;
+    }
+    _NODISCARD constexpr year_month_weekday_last operator/(const year_month& _Ym, const weekday_last& _Wdl) noexcept {
+        return {_Ym.year(), _Ym.month(), _Wdl};
+    }
+    _NODISCARD constexpr year_month_weekday_last operator/(
+        const year& _Year, const month_weekday_last& _Mwdl) noexcept {
+        return {_Year, _Mwdl.month(), _Mwdl.weekday_last()};
+    }
+    _NODISCARD constexpr year_month_weekday_last operator/(int _Year, const month_weekday_last& _Mwdl) noexcept {
+        return year{_Year} / _Mwdl;
+    }
+    _NODISCARD constexpr year_month_weekday_last operator/(
+        const month_weekday_last& _Mwdl, const year& _Year) noexcept {
+        return _Year / _Mwdl;
+    }
+    _NODISCARD constexpr year_month_weekday_last operator/(const month_weekday_last& _Mwdl, int _Year) noexcept {
+        return year{_Year} / _Mwdl;
+    }
+
+    // Calendrical constants

```

Comment:
Should this comment be removed? Or made SHOUTY?

Reply:
I found a single `// CONSTANTS` comment in `<random>`, so I'm not entirely sure.  I'll leave this for another maintainer to chime in on.